src/main/java/study/step2/StringAddCalculator.java

```diff
@@ -0,0 +1,54 @@
+package study.step2;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class StringAddCalculator {
+
+    static int splitAndSum(String text) {
+
+        if (text == null) return 0;
+        if (text.isEmpty()) return 0;
+
+        // java.util.regex 패키지의 Matcher, Pattern import
+        Matcher m = Pattern.compile("//(.)\n(.*)").matcher(text);
+        if (m.find()) {
+            String customDelimiter = m.group(1);
+            String[] tokens = m.group(2).split(customDelimiter);
+
+            return sum(tokens);
+        } else {
+            return sum(split(text));
+        }
+
+    }
```

Comment:
```suggestion
            return sum(tokens);
        } 
        return sum(split(text));
    }
```

앞에서 return을 했는데 이후 else 라는 키워드가 필요할까요?
이번 과정에서 불필요한 else 블록을 제거해보는 습관을 가져볼까요?